content/guides/reader_conditionals.adoc

```diff
@@ -0,0 +1,198 @@
+= Reader Conditionals Guide
+Daniel Compton
+2015-11-17
+:jbake-type: page
+:toc: macro
+
+ifdef::env-github,env-browser[:outfilesuffix: .adoc]
+
+toc::[]
+
+== Introduction
+
+Reader conditionals are a feature added in Clojure 1.7. They are designed to allow different dialects of Clojure to share common code that is mostly platform independent, but contains some platform dependent code. If you are writing code across multiple platforms that is mostly independent you should use `.clj` and `.cljs` for this.
+
+Reader conditionals are integrated into the Clojure Compiler, and don't require any extra tooling beyond Clojure 1.7 or greater. To use Reader conditionals, all you need is for your file to have a `.cljc` extension and to use Clojure 1.7 or ClojureScript 0.0-3196 or higher. Reader conditionals are data, and can be manipulated like ordinary Clojure data. For more technical details, see the reference page on <<xref/../../reference/reader#,the reader>>.
+
+There are two types of reader conditionals, standard and splicing. The standard conditional reader behaves similarly to a traditional `cond`. The syntax for usage is `#?` and looks like:
+
+[source,clojure]
+----
+#?(:clj  (Clojure expression)
+   :cljs (ClojureScript expression)
+   :clr  (Clojure CLR expression))
+----
+
+The syntax for a splicing conditional read is `#?@`. It is used to splice lists into the containing form. So the Clojure reader would read this:
```

Comment:
Would it be possible to paraphrase the concept of *splicing* here? I for one did not know what it meant when I first learned clojure.